index.html

```diff
@@ -1,6 +1,6 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="ja">
-  <head>
+  <meta>
```

Comment:
ここはheadのままにすべきな気がします
```suggestion
  <head>
```

Reply:
あれ、ほんとだ
なんかのタイミングで書き換わっちゃったっぽい